unity_jobs_simulator/Assets/Scripts/Robot_Controller.cs

```diff
@@ -135,6 +129,14 @@ void FixedUpdate()
         }
         else // Otherwise, the robot is incrementing the angles to the next incremental position
         {
+            List<bool> anglesReached = SameAngles(_nextAngles, TargetJointAngles); // We check to see if some of the angles have already arrived
```

Comment:
This name is very similar to "SamePos" so might confuse someone. This function is also only used once right here and feels like it is custom for this case. It would be cleaner to simply have the code here, instead of using a function for this. 

In the interest of getting this merged, can you add a TODO in a comment to move the function logic here?